PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!--- PR Checklist --> 
+<!--- Did you: -->
+<!--- * Check your works conforms to the **OpenMRS 3.0 Styleguide** (at [om.rs/styleguide](https://om.rs/styleguide)? -->
+<!--- * Include screenshots or gifs of UI changes? --> 
+<!--- * Check for feature overlap with **existing widgets** at [om.rs/directory](https://om.rs/directory)? -->
+<!--- * Link to the issue (e.g. at https://issues.openmrs.org/browse/____ -->
+<!--- * Check all new and existing **tests passed**? (It is your responsibility to make sure your code works.) --> 
```

Comment:
About the tests: We could skip this since we already have the pre-commit hooks and GitHub Actions pipeline for running the tests and failing the PR when something is wrong in that area. Also, I already had situations where the tests run fine locally and then fail in the pipelines. -> They should be the "source of truth" for tests anyway.

Maybe we could replace it with a hint about how to format the code? That's something that new contributors probably don't know about and which ultimately fails the pipelines during the linting stage. We could then even describe how to do that (i.e. run `yarn prettier` in the monorepos before creating the PR).